crowbar_framework/spec/lib/runtime_checks_spec.rb

```diff
@@ -0,0 +1,107 @@
+#
+# Copyright 2015, SUSE LINUX Products GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "spec_helper"
+require "logger"
+
+describe RuntimeChecks::NetworkChecks do
+  RuntimeChecks::NetworkChecks.stubs(:hostname).returns("crowbar")
```

Comment:
I'm not sure if that is correct to write that subbing stuff within the describe blocks directly.